apps/req.c

```diff
@@ -1058,7 +1058,6 @@ int req_main(int argc, char **argv)
     BIO_free(addext_bio);
     BIO_free_all(out);
     EVP_PKEY_free(pkey);
-    EVP_MD_free(md_alg);
```

Comment:
Looking at `digest` and `md_alg`, it looks to me like one of them is redundant and can be removed completely.